src/main/asciidoc/batch/file-ingest-sftp/local.adoc

```diff
@@ -0,0 +1,209 @@
+[[sftp-file-ingest-local]]
+==== Using the Local Server
+
+===== Additional Prerequisites
+
+* A running local Data Flow Server
+include::{docs_dir}/local-server.adoc[]
+
+* Running instance of link:http://kafka.apache.org/downloads.html[Kafka]
+* Either a remote or local host accepting SFTP connections.
+
+* A database tool such as link:https://dbeaver.jkiss.org/download/[DBeaver] to inspect the database contents
+
+NOTE: To simplify the dependencies and configuration in this example, we will use our local machine acting as an SFTP server.
+
+===== Building and Running the Demo
+
+. Build the demo JAR
++
+From the root of this project:
++
+```
+$ cd batch/file-ingest
+$ mvn clean package
+```
+NOTE: For convenience, you can skip this step.
+The jar is published to the  https://repo.spring.io/libs-snapshot-local/io/spring/cloud/dataflow/ingest/ingest/1.0.0.BUILD-SNAPSHOT/[Spring Maven repository]
```

Comment:
Should we link to the absolute path of the app at: https://repo.spring.io/libs-snapshot-local/io/spring/cloud/dataflow/ingest/ingest/1.0.0.BUILD-SNAPSHOT/ingest-1.0.0.BUILD-SNAPSHOT.jar?

Reply:
That would just download it I think.